### PR TITLE
Add sqlite documentation

### PR DESCRIPTION
**Pull request by kren1**
Should be merged if klee/klee#1025 ever gets in.

### CODE REVIEW

**Comment by 251** [comment]
```
L27: -the
L32: -therefore
L35: +to (keep)
L36: rephrase: and to test multiple...
L64: -however
L67: needs the [...] +"cmake flags" (or remove "the")?
L83: [P]ython
L92: Counterexample Cache (instead of "solver")
L94+: rephrase, hard to read
L99: runtime[s]
L99+: better explanation or examples (uclibc, posix runtime, ...)
L175: Since LLVM 5.0 -> For LLVM versions...
L176: Therefore[,]
L250: However[,] since we use SQLite3[,] a

Could you please also fix:
L173:
... a temporary file (``%t1.bc``).
Then KLEE is executed on this generated temporary file.
If either program returns a non-zero exit code (or crashes) then test is considered to have failed.

-> to sth. like

... a temporary bitcode file (``%t1.bc``).
Then, KLEE symbolically executes this bitcode file with one of its runtimes (here ``--libc=klee``).
If the program returns a non-zero exit code or crashes, the test is considered to have failed.

L237: _KLEE_RUNTIME_LIBRARY_PATH_ -> ``KLEE_RUNTIME_LIBRARY_PATH``
```

**Comment by kren1** [comment]
Applied the suggestions in both PRs and split them

File: docs/developers-guide.md [hunk 1]
```diff
@@ -169,3 +169,23 @@ KLEE uses LLVM's CommandLine library for adding options to tools in KLEE, which
 ### Run-time libraries
 
 KLEE searches for run-time libraries in install and build paths. These are hard-coded to the binary, so if the filesystem tree changes, KLEE will not find them until recompiled. This behaviour can be overridden by setting _KLEE_RUNTIME_LIBRARY_PATH_ environment variable to the path to the libraries.
+
+### KLEE statistics
+
+
+KLEE uses [SQLite3](https://www.sqlite.org) to store the statistics of it runs
```
Comment (251) [inline docs/developers-guide.md]: its

**Comment by 251** [comment]
Maybe we should add a description for Grafana?

**Comment by kren1** [comment]
That's a good point

**Comment by kren1** [comment]
@251 done, this shouldn't be merged before klee/klee#1086 because grafana doesn't work properly at the moment.

**Comment by 251** [comment]
Thanks @kren1!

**Pull request merged**